src/CattyTests/Resources/ParserTests/Custom/ValidProjectAllBricks/ValidProjectAllBricks0991.xml

```diff
@@ -247,6 +247,15 @@
                       <brick type="NextLookBrick">
                           <commentedOut>false</commentedOut>
                       </brick>
+                      <brick type="SetLookByIndexBrick">
```

Comment:
Please also add Brick to `ValidProjectAllBricks0992.xml` and `ValidProjectAllBricks0993.xml`.